### PR TITLE
Update Polish (pl) Use Cases buckets

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update Polish (pl) Use Cases buckets

### CODE REVIEW

File: src/content/translations/pl/desci/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+title: Zdecentralizowana nauka (DeSci)
+description: Przegląd zdecentralizowanej nauki na Ethereum
+lang: pl
+template: use-cases
+emoji: ":microscope:"
+sidebarDepth: 2
+image: ../../../../assets/future_transparent.png
+alt: ""
```
Comment (nhsz) [inline src/content/translations/pl/desci/index.md:9]: we should have an `alt` value

File: src/content/translations/pl/decentralized-identity/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+---
+title: Zdecentralizowana tożsamość
+description: Czym jest zdecentralizowana tożsamość i dlaczego jest ważna?
+lang: pl
+template: use-cases
+emoji: ":id:"
+sidebarDepth: 2
+image: ../../assets/eth-gif-cat.png
```
Comment (nhsz) [inline src/content/translations/pl/decentralized-identity/index.md:8]: we should have an `alt` value

File: src/content/translations/pl/social-networks/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
+title: Zdecentralizowane sieci społecznościowe
+description: Przegląd zdecentralizowanych serwisów społecznościowych w Ethereum
+lang: pl
+template: use-cases
+emoji: ":mega:"
+sidebarDepth: 2
+image: ../../../../assets/ethereum-learn.png
```
Comment (nhsz) [inline src/content/translations/pl/social-networks/index.md:8]: we should have an `alt` value

**Review by nhsz — COMMENTED** [review]
LGTM in general, noticing we're missing `alt` value on some articles

File: src/content/translations/pl/desci/index.md (hunk 1 above)
Comment (minimalsm) [reply src/content/translations/pl/desci/index.md:9]: I actually think we have a lot of alt tags in places we shouldn't 🤔 Alt tags shouldn't be used on decorative images: https://www.w3.org/WAI/tutorials/images/decorative/

Comment (nhsz) [reply src/content/translations/pl/desci/index.md:9]: Mh, interesting, we could remove `alt` from this file then

File: src/content/translations/pl/decentralized-identity/index.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+- [Co to jest zdecentralizowana tożsamość i dlaczego warto się nią zajmować?](https://web3.hashnode.com/what-is-decentralized-identity) — _Emmanuel Awosika_
+
+### Materiały wideo {#videos}
+
+- [Zdecentralizowana tożsamość (oraz dodatkowa sesja transmisji na żywo)](https://www.youtube.com/watch?v=ySHNB1za_SE&t=539s) — _świetny film szkoleniowy na temat zdecentralizowanej tożsamości autorstwa Andreasa Antonopoulosa._
+- [Logowanie za pomocą Ethereum i zdecentralizowanej tożsamości za pomocą Ceramic, IDX, React i 3ID Connect](https://www.youtube.com/watch?v=t9gWZYJxk7c) — _samouczek YouTube pokazujący, jak zbudować system zarządzania tożsamością do tworzenia, odczytywania i aktualizowania profilu użytkownika przy użyciu portfela Ethereum autorstwa Nadera Dabita._
+- [BrightID — zdecentralizowana tożsamość w Ethereum](https://www.youtube.com/watch?v=D3DbMFYGRoM) — _odcinek podcastu Bankless omawiający BrightID, zdecentralizowane rozwiązanie tożsamości dla Ethereum._
+- [Internet poza łańcuchem: zdecentralizowana tożsamość i weryfikowalne dane uwierzytelniające](https://www.youtube.com/watch?v=EZ_Bb6j87mg) — prezentacja EthDenver 2022 autorstwa Evina McMullena.
```
Comment (corwintines) [inline src/content/translations/pl/decentralized-identity/index.md:178]:
```suggestion
- [Internet poza łańcuchem: zdecentralizowana tożsamość i weryfikowalne dane uwierzytelniające](https://www.youtube.com/watch?v=EZ_Bb6j87mg) — _prezentacja EthDenver 2022 autorstwa Evina McMullena._
```

File: src/content/translations/pl/desci/index.md (hunk 1 above)
Comment (nhsz) [inline src/content/translations/pl/desci/index.md:9]:
```suggestion
```

**Review by nhsz — APPROVED** [review]

**Review by corwintines — APPROVED** [review]

**Pull request merged**